main.nf

```diff
@@ -20,21 +20,20 @@ def helpMessage() {
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/epitopeprediction -profile <docker/singularity/conda/institute> --input "*.vcf.gz"
+<<<<<<< HEAD
+    nextflow run nf-core/epitopeprediction --input '*.vcf.gz' -profile docker
 
     Mandatory arguments:
-      -profile [str]                        Configuration profile to use. Can use multiple (comma separated)
-                                            Available: conda, docker, singularity, test, awsbatch, <institute> and more
-
-    Input:
       --input [file]                        Path to input data (must be surrounded with quotes)
```

Comment:
This could be confusing as it seems as if the `--input` parameter is mandatory, but it just has to be one of `--input`, `--peptides` or `--proteins`.